pandas/io/tests/test_packers.py

```diff
@@ -543,25 +543,6 @@ def test_sparse_frame(self):
         self._check_roundtrip(ss3, tm.assert_frame_equal,
                               check_frame_type=True)
 
-    def test_sparse_panel(self):
-
```

Comment:
hmm I think we have legacy pickled / msgpacks that DO serialize this, so need to ignore that NotImplementedError when checking (IOW in io/tests/test_pickle.py)